examples/policies/l3/fqdn/fqdn.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: "to-fqdn"
+spec:
+  endpointSelector:
+    matchLabels:
+      app: test-app
+  egress:
+    - toEndpoints:
```

Comment:
In the beginning this was confused to me, I thought we were only be able to connect with kube-dns on `my-remote-service.com`. My confusion only happened because the json file above is does not exactly translates to this policy.

Reply:
Apart from the k8s stuff and kube-dns labels, how doesn't it? The yaml has 2 egress rules, and one allows the DNS lookups themselves. The json has two egress rules, and one allows the DNS lookups as well. The problem is that I can't make up a label that is probably correct for the json case since I have no idea what the DNS container would be labelled as. In the k8s case it's a reasonable guess.
Still, in both cases, I wanted the example to be almost directly usable. I could remove the dns parts, since they aren't strictly part of the ToFQDNs part. No one reads docs, though, so this was an attempt to make it clear that they also need to allow the DNS lookups themselves.